content/de/includes/task-tutorial-prereqs.md

```diff
@@ -0,0 +1,8 @@
+You need to have a Kubernetes cluster, and the kubectl command-line tool must
```

Comment:
It is better to not have English content present as it may change while you're translating.